controls/radwindow/features/window-icon.md

```diff
@@ -53,16 +57,38 @@ If you want to share a common icon layout structure for the content of multiple
 	</UserControl.Resources>
 {{endregion}}
 
-#### __C#__
+#### __[C#] Setting the IconTemplate property in code-behind__
 
 {{region cs-radwindow-features-window-icon_3}}
 	RadWindow radWindow = new RadWindow();
 	radWindow.IconTemplate = this.Resources["WindowIconTemplate"] as DataTemplate;
 {{endregion}}
 
-#### __VB.NET__
+#### __[VB.NET] Setting the IconTemplate property in code-behind__
 
 {{region vb-radwindow-features-window-icon_4}}
 	Dim radWindow As New RadWindow()
 	radWindow.IconTemplate = TryCast(Me.Resources("WindowIconTemplate"), DataTemplate)
+{{endregion}}
+
+## IconMargin Property
+
+Through the `IconMargin` property of the `RadWindow`, you can control the `Margin` that is applied to the chosen `Icon`.

```

Comment:
Most of the time, it is better to switch the order of the information in the sentence. This is valid for documentation, bug/feature titles and probably some other places.

Currently, the sentence tells that you need to set the property in order to get the desired result. The opposite order is easier to read and you can determine (early) if you should finish the whole sentence or the information is not sufficient. For example, you can tell something like "to get this result, set this property".